YMChat/src/main/res/values/strings.xml

```diff
@@ -7,7 +7,6 @@
     <string name="ym_lite_chatbot_base_url">file:///android_asset/yellow-index-lite.html</string>
     <string name="ym_fab_voice_button">voice button</string>
     <string name="application_id_for_provider">com.yellowmessenger.ymchatexample.fileprovider</string>
-    <string name="ym_file_provider">@string/application_id_for_provider</string>
```

Comment:
We decided to keep this string. Was that something else?

Reply:
that was above one..this we use internally

Reply:
that was above one..this we used internally